scripts/activatePlayground.js

```diff
@@ -28,33 +28,40 @@ YellowBox.ignoreWarnings([
 ]);
 
 // import App from './app/App';
-import App from './app/playground/src/Navigation';
+import App from './packages/playground/src/Navigation';
 
 AppRegistry.registerComponent('reactNativeApp', () => App);
 `;
 
 fs.writeFileSync(indexFilePath, indexFileContent);
 
-// conviently enough, find sync returs a promise
-findSync('PlaygroundRenderer.render', 'app')
-  .then(files => {
-    return Object.keys(files);
-  })
-  .then(playgroundTestFiles => {
-    const playgroundImportsFileContent = [
-      '// @flow',
-      '',
-      playgroundTestFiles
-        .map(
-          // Replacing app from filepath with ../.. which is the path from playgroundList component to app folder
-          item => `import '${item.replace('app', '../..')}';`,
-        )
-        .join(os.EOL),
-      '',
-    ].join(os.EOL);
-
-    fs.writeFileSync(playgroundImportsFilePath, playgroundImportsFileContent);
-
-    // Can also be started on android with yarn playground android
-    execSync(`yarn ${process.argv[2] || 'ios'}`, { stdio: 'inherit' });
-  });
+let playgroundTestFiles = [];
+
+const appPromise = findSync('PlaygroundRenderer.render', 'app').then(files => {
+  playgroundTestFiles = playgroundTestFiles.concat(Object.keys(files));
+});
+
+const packagePromise = findSync('PlaygroundRenderer.render', 'packages').then(
+  files => {
+    playgroundTestFiles = playgroundTestFiles.concat(Object.keys(files));
+  },
+);
+
+Promise.all([appPromise, packagePromise]).then(() => {
+  const playgroundImportsFileContent = [
+    '// @flow',
+    '',
+    playgroundTestFiles
+      .map(
+        // Replacing app from filepath with ../.. which is the path from playgroundList component to 'packages' folder
```

Comment:
This comment does not seem to match what is happening

Reply:
Fixed. I would appreciate if you could just fix these minor issues and typos if you are trying it locally. You are a merge master - you can do it... 🙂

Reply:
Sure, makes sense, I will do it for the next ones 👍